dynawo/sources/Models/Modelica/Dynawo/Electrical/Injectors/InjectorIDQ.mo

```diff
@@ -58,20 +58,20 @@ protected
   parameter Types.ComplexApparentPowerPu s0Pu  "Start value of apparent power at injector terminal in p.u (base SnRef) (receptor convention)";
   parameter Types.ComplexCurrentPu i0Pu  "Start value of complex current at injector terminal in p.u (base UNom, SnRef) (receptor convention)";
     
-  parameter Types.PerUnit Id0Pu "Start value of id in p.u (base SNom)";
-  parameter Types.PerUnit Iq0Pu "Start value of iq in p.u (base SNom)";
+  parameter Types.CurrentModulePu Id0Pu "Start value of id in p.u (base SNom)";
+  parameter Types.CurrentModulePu Iq0Pu "Start value of iq in p.u (base SNom)";
       
 equation
 
   UPhase = ComplexMath.arg(terminal.V);
   UPu = ComplexMath.'abs'(terminal.V);
-  uPu = terminal.V;
+  uPu = terminal.V; // The variable uPu is used in stead of a terminal connection for components that need the voltage phasor.
```

Comment:
It may be better to delete this comment and write it in the declaration of the variable instead, and mention that terminal is for network connections only, i.e.

ComplexBlocks.Interfaces.ComplexOutput uPu (re(start = u0Pu.re), im(start=u0Pu.im)) "complex inverter terminal voltage, used instead of terminal.V for control components that need the voltage phasor, terminal variable being used for network connections only"